README.md

```diff
@@ -1,75 +1,88 @@
 
 ### 2. Launching Prisma/DB Demo Suite
 
 To launch the demo suite you will need to open your command line interface. On Windows, you may launch PowerShell; on Mac, you can launch Terminal app; if you are on Linux, you sure know what to do :)
 
-**1)** Pull the latest image of MySQL with Prisma/DB libraries:
-
-`> docker pull bazzilic/prismadb-mysql`
+**1)** Obtain the Docker Compose file:
 
-And launch it, setting the root password to `toor`:
+Download/save the following YAML Docker Compose file to your machine:
+https://github.com/PrismaDB/PrismaDB/blob/master/scripts/postgres/docker-compose/basic/linux/docker-compose.yml ([Raw File](https://raw.githubusercontent.com/PrismaDB/PrismaDB/master/scripts/postgres/docker-compose/basic/linux/docker-compose.yml))
 
-`> docker run -d --rm -p 3306:3306 --name prismadb-db -e MYSQL_ROOT_PASSWORD=toor bazzilic/prismadb-mysql`
+This Docker Compose file will start a PostgreSQL server with Prisma/DB UDFs, as well as the Prisma/DB Proxy (trial version) for PostgreSQL. Importing/exporting of settings are disabled in the trial build of Prisma/DB Proxy, but other than that, it is a fully featured build.
 
-Note, that it takes about 30 seconds before the db server starts accepting connections.
+**2)** Run the containers in detached mode:
 
-**2)** Pull the latest image of Prisma/DB Proxy
+`> docker-compose up -d`
 
-`> docker pull bazzilic/prismadb-proxy-mysql`
+**3)** Launch Psql command line tool to connect to the database through the Prisma/DB Proxy:
 
-And launch it as well, linking it to the database container.
+`> docker exec -it prismadb_postgres_db psql -U postgres -h prismadb_postgres_proxy -p 4000 testdb`
 
-`> docker run --rm -d --name prismadb-proxy -p 4000:4000 --link=prismadb-db -e MYSQL_ADDRESS=prismadb-db bazzilic/prismadb-proxy-mysql`
-
-**3)** Launch MySQL command line tool to connect to the database through the Prisma/DB Proxy
-
-`> docker run -it --rm --link=prismadb-proxy mysql mysql -h prismadb-proxy -P 4000 -u root -ptoor`
+Use password `Qwer!234` to authenticate as `postgres` user, and enter `\q` anytime to quit.
 
 **4)** Use some of the following queries to try out the encrypted database:
 
 ```SQL
 CREATE TABLE t1
 (
-	a INT ENCRYPTED FOR (INTEGER_MULTIPLICATION, INTEGER_ADDITION, SEARCH),
-	b INT ENCRYPTED FOR (INTEGER_MULTIPLICATION, INTEGER_ADDITION, SEARCH),
+	a INT ENCRYPTED FOR (MULTIPLICATION, ADDITION, RANGE),
+	b INT ENCRYPTED FOR (MULTIPLICATION, ADDITION, RANGE),
 	c INT,
 	d VARCHAR(30) ENCRYPTED FOR (STORE, SEARCH),
 	e VARCHAR(30)
 );
 
+-- Display column details
+SHOW COLUMNS FROM t1;
+
 INSERT INTO t1 (a, b, c, d, e) VALUES
 ( 1,  2,   3, 'Hello', 'Prisma/DB'),
 (12,  0,   7, 'Test', 'data'),
 ( 0,  2, 123, 'This is encrypted', 'And this is not'),
 (71, 67,  13, 'Last', 'row');
 
 -- Arithmetic operations and search over encrypted values
-SELECT a, b, a + b AS `SUM`, b * a AS `MUL`, (b * a) + b AS `EXPR`
+SELECT a, b, a + b, b * a, (b * a) + b, c, d, e
```

Comment:
Have you checked that the syntax in this sample queries is compatible with Postgres? It was initially written for MySQL

Reply:
Yup, they are.